lib/apiservers/engine/backends/container.go

```diff
@@ -1032,11 +1053,18 @@ func (c *Container) ContainerTop(name string, psArgs string) (*types.ContainerPr
 
 // Containers returns the list of containers to show given the user's filtering.
 func (c *Container) Containers(config *types.ContainerListOptions) ([]*types.Container, error) {
+	defer trace.End(trace.Begin(fmt.Sprintf("ListOptions %#v", config)))
```

Comment:
we can -- I'd propose that we have a large discussion about it.  Specifically identifying a common approach for the utility functions.  For example we are converting to docker specific formats in many cases...that might be a struct, string, etc..I'd think we can centralize those in a package etc...

So how about we create an issue for cleaning up the persona -- we could do a research spike and then nail down tasks / deliverables -- thoughts?